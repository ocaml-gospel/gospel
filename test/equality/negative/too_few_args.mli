val f : int -> bool
(*@ equality *)
