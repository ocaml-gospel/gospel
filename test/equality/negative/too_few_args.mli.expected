
*******************************
********** Parsed file ********
*******************************
val f : int -> bool[@@gospel {| equality |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

val f : int -> bool
(*@ equality
     *)
File "too_few_args.mli", lines 1-2, characters 0-25:
Error: Type checking error: type is incompatible with equality clause
